test_generators/ssz/renderers.py

```diff
@@ -0,0 +1,93 @@
+from collections.abc import (
+    Mapping,
+    Sequence,
+)
+
+from eth_utils import (
+    encode_hex,
+    to_dict,
+)
+
+from ssz.sedes import (
+    BaseSedes,
+    Boolean,
+    Bytes,
+    BytesN,
+    Container,
+    List,
+    UInt,
+)
+
+
+def render_value(value):
+    if isinstance(value, bool):
+        return value
+    elif isinstance(value, int):
+        return str(value)
+    elif isinstance(value, bytes):
+        return encode_hex(value)
+    elif isinstance(value, Sequence):
+        return tuple(render_value(element) for element in value)
+    elif isinstance(value, Mapping):
+        return render_dict_value(value)
+    else:
+        raise ValueError(f"Cannot render value {value}")
+
+
+@to_dict
+def render_dict_value(value):
+    for key, value in value.items():
+        yield key, render_value(value)
+
+
+def render_type_definition(sedes):
+    if isinstance(sedes, Boolean):
+        return "bool"
+
```

Comment:
the newline between `if` and `elif` is pretty nonstandard for python

Reply:
See https://github.com/ethereum/eth2.0-test-generators/blob/bbcfb29d4864d802890d440ba5fa998e70fc2b4b/ssz/renderers.py#L44
Think the same, but you could consider it somewhat more readable. It's changing anyway with the new SSZ tests, so I leave it as-is for now.